From 7df7c79fbb76975768e308f224b0b40c81b598a2 Mon Sep 17 00:00:00 2001
From: Kalle Lampila <kallex.lampila@intel.com>
Date: Wed, 23 Jan 2013 14:58:08 +0200
Subject: [PORT FROM MAIN] camera: disable shutter sound when entering record mode

BZ: 63323 72052

This disables shutter sound when entering record mode. This is needed
because after burst mode picture capture shutter sound is not disable
like it is make after normal capture.

DO NOT UPSTREAM.

Change-Id: I4ec3e0bd442ac49657340e6bba752717e8e80782
Orig-Change-Id: I34d77f891c25280ba51cabdcdf7707f55cd7650e
Signed-off-by: Kalle Lampila <kallex.lampila@intel.com>
Reviewed-on: http://android.intel.com:8080/88218
Reviewed-by: cactus <cactus@intel.com>
Reviewed-by: Hu, Gang A <gang.a.hu@intel.com>
Reviewed-by: Verdu, Miguel <miguel.verdu@intel.com>
Reviewed-by: Yang, Liang L <liang.l.yang@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Koski, Anttu <anttu.koski@intel.com>
Reviewed-by: buildbot <buildbot@intel.com>
Tested-by: buildbot <buildbot@intel.com>
---
 services/camera/libcameraservice/CameraClient.cpp |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/services/camera/libcameraservice/CameraClient.cpp b/services/camera/libcameraservice/CameraClient.cpp
index e577fa3..9c1a504 100644
--- a/services/camera/libcameraservice/CameraClient.cpp
+++ b/services/camera/libcameraservice/CameraClient.cpp
@@ -356,6 +356,7 @@ status_t CameraClient::startPreview() {
 // start recording mode
 status_t CameraClient::startRecording() {
     LOG1("startRecording (pid %d)", getCallingPid());
+    disableMsgType(CAMERA_MSG_SHUTTER);
     return startCameraMode(CAMERA_RECORDING_MODE);
 }
 
-- 
1.7.4.1

